CMakeLists.txt

```diff
@@ -65,8 +65,11 @@ include(CMakeDependentOption)
 option(BUILD_DAEMON "Build bitcoind executable." ON)
 option(BUILD_GUI "Build bitcoin-qt executable." OFF)
```

Comment:
Currently, in `master`, we build `bitcoin-qt` by default, what's the reasoning behind this to be changed?

Reply:
During our calls, we discussed the concept of build options. We have reached a rough consensus to make them "opt-in" rather than "opt-out" as much as possible.